masterlist.yaml

```diff
@@ -822,9 +822,12 @@ groups:
   - name: &horizonGroup Horizon
     description: 'A group for Horizon modules that need to be sorted late.'
     after: [ *scrapGroup ]
+  - name: &frostGroup FROST
+    description: 'A group of Horizon modules that need to be sorted late.'
+    after: [ *horizonGroup ]
 
   - name: &lightingGroup Lighting
-    after: [ *horizonGroup ]
+    after: [ *frostGroup ]
```

Comment:
There's not much point adding a new group if it's not going to be used.
Horizon and Frost are both total conversion mods, which shouldn't be used together, so they could use the same group.
It would probably be easiest to remove both the Horizon and Frost groups, and add a new group to handle both.

Reply:
I might just remove this edit, FROST submods usually don't need special priorities I have found so far.